stix_shifter_modules/elastic_ecs/stix_translation/json/to_stix_map.json

```diff
@@ -232,6 +468,816 @@
+      },
+      "not_after": {
+        "key": "x509-certificate.validity_not_after",
+        "object": "client"
+      },
+      "not_before": {
+        "key": "x509-certificate.validity_not_before",
+        "object": "client"
+      },
+      "subject": {
+        "key": "x509-certificate.subject",
+        "object": "client"
+      }
+    },
+    "server": {
+      "issuer": {
+        "key": "x509-certificate.issuer",
+        "object": "server"
+      },
+      "hash": {
+        "sha256": {
+          "key": "x509-certificate.hashes.SHA-256",
+          "object": "server"
+        },
+        "sha1": {
+          "key": "x509-certificate.hashes.SHA-1",
+          "object": "server"
+        },
+        "md5": {
+          "key": "x509-certificate.hashes.MD5",
+          "object": "server"
+        }
+      },
+      "not_after": {
+        "key": "x509-certificate.validity_not_after",
+        "object": "server"
+      },
+      "not_before": {
+        "key": "x509-certificate.validity_not_before",
+        "object": "server"
+      },
+      "subject": {
+        "key": "x509-certificate.subject",
+        "object": "server"
+      }
+    }
+  },
+  "trace": {
+    "id": {
+      "key": "x_com_elastic_ecs_trace.id",
+      "cybox": false
+    }
+  },
+  "transaction": {
+    "id": {
+      "key": "x_com_elastic_ecs_transaction.id",
+      "cybox": false
+    }
+  },
+  "user_agent": {
+    "name": {
+      "key": "x_com_elastic_ecs_user_agent.name",
+      "cybox": false
+    },
+    "original": {
+      "key": "x_com_elastic_ecs_user_agent.original",
+      "cybox": false
+    },
+    "version": {
+      "key": "x_com_elastic_ecs_user_agent.version",
+      "cybox": false
+    },
+    "device": {
+      "name": {
+        "key": "x_com_elastic_ecs_user_agent.device_name",
+        "cybox": false
+      }
+    }
+  },
+  "vulnerability": {
+    "category": {
+      "key": "x_com_elastic_ecs_vulnerability.category",
+      "cybox": false
+    },
+    "classification": {
+      "key": "x_com_elastic_ecs_vulnerability.classification",
+      "cybox": false
+    },
+    "description": {
+      "key": "x_com_elastic_ecs_vulnerability.description",
+      "cybox": false
+    },
+    "enumeration": {
+      "key": "x_com_elastic_ecs_vulnerability.enumeration",
+      "cybox": false
+    },
+    "id": {
+      "key": "x_com_elastic_ecs_vulnerability.id",
+      "cybox": false
+    },
+    "reference": {
+      "key": "x_com_elastic_ecs_vulnerability.reference",
+      "cybox": false
+    },
+    "report_id": {
+      "key": "x_com_elastic_ecs_vulnerability.report_id",
+      "cybox": false
+    },
+    "severity": {
+      "key": "x_com_elastic_ecs_vulnerability.severity",
+      "cybox": false
+    },
+    "scanner": {
+      "vendor": {
+        "key": "x_com_elastic_ecs_vulnerability.scanner_vendor",
+        "cybox": false
+      }
+    },
+    "score": {
+      "base": {
+        "key": "x_com_elastic_ecs_vulnerability.score_base",
```

Comment:
In cases where we have a chain of properties like this, would it be better to write `x_com_elastic_ecs_vulnerability.score.base`, `x_com_elastic_ecs_vulnerability.score.environmental`...